test/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Adds a googletest-based test named for the given source file,
+# linked against Aribter.
+function(add_googletest test_source)
+  get_filename_component(test_name "${test_source}" NAME_WE)
+  add_executable("${test_name}" TestValue.cpp "${test_name}.cpp")
+  target_link_libraries("${test_name}"
+    PRIVATE "${ARBITER_TARGET}" gtest gtest_main)
+  target_include_directories("${test_name}"
+    PRIVATE
+      "${CMAKE_SOURCE_DIR}/external/googletest/googletest/include"
```

Comment:
Is there a way to include this like `-isystem` does, to silence warnings in gtest headers like this?

```
In file included from /Users/jspahrsummers/GitHub/Arbiter/test/VersionTest.cpp:3:
/Users/jspahrsummers/GitHub/Arbiter/external/googletest/googletest/include/gtest/gtest.h:1392:11: warning: comparison of integers of different signs: 'const unsigned int' and 'const int'
      [-Wsign-compare]
  if (lhs == rhs) {
      ~~~ ^  ~~~
/Users/jspahrsummers/GitHub/Arbiter/external/googletest/googletest/include/gtest/gtest.h:1421:12: note: in instantiation of function template specialization
      'testing::internal::CmpHelperEQ<unsigned int, int>' requested here
    return CmpHelperEQ(lhs_expression, rhs_expression, lhs, rhs);
           ^
/Users/jspahrsummers/GitHub/Arbiter/test/VersionTest.cpp:12:3: note: in instantiation of function template specialization 'testing::internal::EqHelper<false>::Compare<unsigned int, int>' requested
      here
  EXPECT_EQ(version._major, 1);
  ^
/Users/jspahrsummers/GitHub/Arbiter/external/googletest/googletest/include/gtest/gtest.h:1924:63: note: expanded from macro 'EXPECT_EQ'
                      EqHelper<GTEST_IS_NULL_LITERAL_(val1)>::Compare, \
                                                              ^
1 warning generated.
```